crypto/s2n_fips.c

```diff
@@ -19,23 +19,33 @@
 
 static int s2n_fips_mode = 0;
```

Comment:
This is now only set, never retrieved or checked?

Reply:
It's returned by `s2n_is_in_fips_mode()` as before. It i still the responsibility of the consumer to initialise the openssl fips module before initialising s2n.